k8s/src/main/scala/io/buoyant/k8s/istio/MixerClient.scala

```diff
@@ -25,14 +26,17 @@ private[telemetry] object MixerClient {
     mapper.readValue[Seq[String]](yaml).asInstanceOf[Seq[String]]
   }
 
-  private[telemetry] def indexOf(dict: Seq[String], str: String): Int =
+  private[this] def indexOf(dict: Seq[String], str: String): Int =
     -1 * (dict.indexOf(str) + 1)
 
-  private[telemetry] def mkReportRequest(): ReportRequest = {
+  // TODO: http1-only for now
+  private[istio] def mkReportRequest(req: Request, rsp: Option[Response]): ReportRequest = {
```

Comment:
yeah! i had thought about moving the body into IstioLogger to make MixerClient not http1-specific, so rather than `logHttp`, we'd have:
```
MixerClient {
  def log(rsp: ResposeRequest)
}
```

my only concern was exposing the IstioLogger to the Mixer protobuf, but perhaps that's better than exposing MixerClient to all the http1-specifics ?